pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3137,16 +3136,25 @@ private void initializeTopicSubscribeRateLimiterIfNeeded(Optional<TopicPolicies>
         }
     }
 
+    private TopicName getPartitionedTopicName() {
```

Comment:
this method looks like an utility method that should be moved to the TopicName class

Reply:
Great point. This is a quite common usage in broker.